src/states/elder/mod.rs

```diff
@@ -231,6 +228,7 @@ impl Elder {
             ignore_candidate_info_counter: 0,
             proving_section_cache: LruCache::with_capacity(PROVING_SECTION_CACHE_SIZE),
             pfx_is_successfully_polled: false,
+            signed_messages_accumulator: BTreeMap::new(),
```

Comment:
Not sure about this name. Why not: `signed_messages`? Calling it "accumulator" makes it sound like it was more than just a simple cache.